RN_Components/src/components/repeat-button/RepeatButton.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+import { TouchableOpacity, View } from 'react-native'
+import Icon from 'react-native-vector-icons/MaterialCommunityIcons'
+
+import { Spacing } from '@theme'
+
+interface IRepeatButtonProp {
+  repeat: boolean
+  setRepeat: Function
+}
+const RepeatButton = (props: IRepeatButtonProp) => {
+  const { repeat, setRepeat } = props
+  return (
+    <View>
+      <TouchableOpacity onPress={() => setRepeat(!repeat)}>
+        {repeat ? (
+          <Icon name="repeat" size={Spacing.space_30} />
+        ) : (
+          <Icon name="repeat-off" size={Spacing.space_30} />
+        )}
```

Comment:
can use condition on name of icon only , we don't have to put it icon as everything else is same , just name is conditional